files/image_config/sysctl/sysctl-net.conf

```diff
@@ -0,0 +1,39 @@
+# All the sysctl for ipv4/ipv6 network.
+# Same will be used in host or docker namespace
+# It should be provided as key=value format for parsing
+net.ipv6.conf.all.disable_ipv6=0
```

Comment:
Why is this global setting ? Its different from the previous setting right?

Reply:
@prsunny yes, because in asic namespace disable_ipv6 is 1 by default and in host it is 0 by default. 
Now making explicitly as 0